src/posts/2019-10-23-SJRK-F2F.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: layouts/post
+title: Stories from the Social Justice Repair Kit
+date: '2019-10-23'
+tags: post
+permalink: posts/2019-10-23-SJRK-F2F.html
+---
+<article class="floe-content floe-news-item">
+                <h2> News </h2>
+                <!-- BEGIN markup for news item -->
+                <h3>Stories from the Social Justice Repair Kit</h3>
+                <time class="floe-date" datetime="2019-10-23">October 23, 2019</time>
+                <p>
+                    On October 17th, the
+                    <a href="https://www.sojustrepairit.org/">Social Justice Repair Kit</a>
+                    project met to reflect on the past three years of work. Participants created
+                    <a href="http://stories.sojustrepairit.org/">stories</a>
+                    to capture their experiences on the project. Highlights included training
+                    youth to lead environmental protection work and doing audio-visual workshops
+                    with youth at the margins, enabling them to not only tell their stories but
+                    also to capture and present them in ways they want to.
+                </p>
+                <p>
+                    <img src="images/SJRK_Drawing.png" alt="Collaborative surprise drawings by the SJRK team." /><br/>
+
+                    At the end of the day, the project team created a collaborative surprise drawing which represents
+                    the creative collaborations that happened throughout the project.
+                </p>
```

Comment:
There's a strange bug here where part of the text content is appearing in a `<pre><code>` block.

![image](https://user-images.githubusercontent.com/564762/87324467-1f0e7580-c4fe-11ea-99f0-b7eab1906f9c.png)

Reply:
Fixed